### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by JeanMarieMineau**
Refactor the code.

### CODE REVIEW

**Comment by JeanMarieMineau** [comment]
Woupsi, it's a mistake.

File: src/main.s [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+; ///////// DISABLE INTERRUPTIONS \\\\\\\\\
+.ORG $0040 				; Write at the address $0040 (vblank interuption)
+	call VBlank
+	reti
+
+.ORG $0100 				; Write at the address $0100 (starting point of the prog)
+	nop							; adviced from nintendo. nop just skip the line.
+	jp start
+; \\\\\\\\\ DISABLE INTERRUPTIONS /////////
```
Comment (g33kex) [inline src/main.s:20]: You're not disabling the interruptions in this sections, but actually handling VBlank interruption, and then handling the program start point.

Comment (JeanMarieMineau) [reply src/main.s:20]: -_-, yep, I'm dumb

**Comment by JeanMarieMineau** [comment]
documentation added

**Pull request merged**